sf-fx-runtime-java-runtime/src/main/java/com/salesforce/functions/jvm/runtime/json/ListParameterizedType.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+package com.salesforce.functions.jvm.runtime.json;
+
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+import java.util.List;
+
+public class ListParameterizedType implements ParameterizedType {
```

Comment:
Does this need a javadoc definition? I don't know what the convention is for when to include them.

Reply:
I took a stab at docs. It looks like most of the javadocs are method docs. I personally prefer a top level class doc. We can talk specifics when Manuel is back.

Reply:
Thanks for taking a stab at docs @schneems! 

In this case, I don't think we really need them here. This class implements a [well documented](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/reflect/ParameterizedType.html) interface of the standard library. The usage of the interface itself is documented there and this class does not do anything fancy and just returns fixed values in each method implementation.

If anyone comes across this class and wonders what it does and what it's for, they can read the `ParameterizedType` docs which are usually just a click away.

Side-note: Standard Javadoc does not support Markdown.